integration-tests/esbuild/aws-sdk.js

```diff
@@ -2,4 +2,4 @@ require('../../').init() // dd-trace
 
 const aws = require('aws-sdk')
 
-void aws.util.inherit
+global.test = aws.util.inherit
```

Comment:
@Qard can you confirm this is intended

Reply:
Yes. The linter yells about the use of void, but without it esbuild will remove the line as dead code. Assigning it to `global.test` prevents that.